modules/dag-shared/src/main/scala/org/tessellation/dag/snapshot/BlockAsActiveTip.scala

```diff
@@ -1,15 +1,20 @@
 package org.tessellation.dag.snapshot
 
 import org.tessellation.dag.domain.block.DAGBlock
+import org.tessellation.ext.cats.data.OrderBasedOrdering
 import org.tessellation.security.signature.Signed
 
-import derevo.cats.{eqv, show}
+import derevo.cats.{order, show}
 import derevo.derive
 import eu.timepit.refined.cats._
 import eu.timepit.refined.types.numeric.NonNegLong
 
-@derive(eqv, show)
+@derive(order, show)
 case class BlockAsActiveTip(
   block: Signed[DAGBlock],
   usageCount: NonNegLong
 )
+
+object BlockAsActiveTip {
+  implicit object Ordering extends OrderBasedOrdering[BlockAsActiveTip]
```

Comment:
It's supposed to be `OrderingInstance` here to avoid naming clash with `scala.math.Ordering`. I will change it with any other PR fixes.